src/bootstrap/builder.rs

```diff
@@ -534,6 +525,18 @@ impl<'a> Builder<'a> {
         Some(help)
     }
 
+    fn get_builder(build: &Build, kind: Kind, paths: Vec<PathBuf>) -> Builder<'_> {
```

Comment:
```suggestion
    fn new(build: &Build, kind: Kind, paths: Vec<PathBuf>) -> Builder<'_> {
```

Will need some changes to call sites as well.

Reply:
There is already a [`Builder::new`](https://github.com/rust-lang/rust/pull/73964/files/b8f10217fe56c3b81894681fda948ccb70ce13a2#diff-4bc384d35e57dc577b9378d4d733d7c3R540). Maybe this could be `new_internal()`?

Reply:
Yeah that seems fine. Would be okay with renaming the Builder::new as well, whatever works for you -- I don't like get_builder as it feels global-y.